test/models/repository_test.rb

```diff
@@ -58,6 +59,19 @@ def test_docker_repository_docker_upstream_name_url
       assert @repo.valid?
     end
 
+    test_attributes :pid => '7967e6b5-c206-4ad0-bcf5-64a7ce85233b'
+    def test_docker_repository_update_name
+      @repo = build(:katello_repository, :docker,
+                    :environment => @library,
+                    :product => katello_products(:fedora),
+                    :content_view_version => @library.default_content_view_version
+                   )
```

Comment:
The test say update but the @repo does not exist yet, wehave a missing @repo.save! here

Reply:
When testing validations, there is no need to save, so i think this is fine.